api/kyverno/v1/image_verification_types.go

```diff
@@ -100,9 +100,13 @@ type AttestorSet struct {
 
 type Attestor struct {
 
-	// StaticKey is a set of attributes used to verify an X.509 public key
+	// Keys specifies one or more public keys
 	// +kubebuilder:validation:Optional
-	StaticKey *StaticKeyAttestor `json:"staticKey,omitempty" yaml:"staticKey,omitempty"`
+	Keys *StaticKeyAttestor `json:"keys,omitempty" yaml:"keys,omitempty"`
```

Comment:
Just to check - these replaced tags are never released, correct?

Reply:
yes, they were added for this release so we do not need to maintain compatibility